firefox-ios/Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController+TabScrollHandlerDelegate.swift

```diff
@@ -53,7 +54,6 @@ extension BrowserViewController: TabScrollHandler.Delegate,
     /// container's mask view to ensure proper visual effects during toolbar animations and state
     /// transitions. Only executes when the toolbar translucency refactor feature flag is enabled.
     private func updateToolbarTranslucency() {
-        guard isToolbarTranslucencyRefactorEnabled else { return }
```

Comment:
We need the flag here to make sure it executes only when the translucency refactor is on.

Reply:
but in the tab scroll refactor version without this flag enabled the implementation looks broken